doc/source/user_guide/io.rst

```diff
@@ -1577,19 +1577,20 @@ value will be an iterable object of type ``TextFileReader``:
 
 .. ipython:: python
 
-   reader = pd.read_csv("tmp.sv", sep="|", chunksize=4)
-   reader
-
-   for chunk in reader:
-       print(chunk)
+   with pd.read_csv("tmp.sv", sep="|", chunksize=4) as reader:
+       reader
+       for chunk in reader:
+           print(chunk)
 
+.. versionchanged:: 1.2
+  ``read_csv/json/sas`` return a context-manager when iterating through a file.
```

Comment:
I *think* you need a blank after the versionadded to make this render correctly, though never remember